src/store/reducers/appReducer.js

```diff
@@ -137,7 +137,7 @@ const updateThemeReducer = (state, { payload }) => update(state,
 
 const logoReducer = (state, { payload, fromAction }) => update(state, {
     logo: { $set: payload.logo ? payload.logo : svgs.contoso },
-    name: { $set: payload.name ? payload.name : 'Contoso' },
+    name: { $set: payload.name ? payload.name : 'companyName' },
```

Comment:
I'm confused, What should be our default name be companyName or Contoso?

Reply:
companyName is the key to translate to Contoso (can't call translate directly from appReducer). In the UI this will show as Contoso